### DISCUSSION TITLE
已交费2年专业版未能成功续期

### DISCUSSION THREAD

**Discussion by ArvieWang** [getlantern/lantern-forum-cn#1000] [General]
![lantern 付款记录Image_20241027151218](https://github.com/user-attachments/assets/3b2b12dd-125f-4539-b04b-521831d3f492)
如上图所示，于2024年9月24日充值2年蓝灯专业版，但是一直提示即将到期，未成功充值，
邮箱：359264293@qq.com
订单号：2024092422001467311438056063
商家订单号：2024092409411995523

**Comment by ArvieWang** [comment]
如果能处理请联系邮箱：359264293@qq.com,并发送新的激活码以便成功登录；
若不能处理，请退款，谢谢！
![lantern 付款记录Image_20241027151218](https://github.com/user-attachments/assets/9006a72f-445f-4cf7-afc9-cac75d20fa39)

如上图所示，于2024年9月24日充值2年蓝灯专业版，但是一直提示即将到期，未成功充值，
邮箱：359264293@qq.com
订单号：2024092422001467311438056063
商家订单号：2024092409411995523

**Comment by MaxMKMax** [comment]
@ArvieWang 您好. 好的, 我帮您提交上去处理问题. 请您稍等我们几天.

**Reply by ArvieWang** [reply]
[Mike777Khoo

> @ArvieWang 您好. 好的, 我帮您提交上去处理问题. 请您稍等我们几天.

感谢，无法正常工作，请尽快帮忙处理，以便推进我们的工作开展！

**Reply by ArvieWang** [reply]
> @ArvieWang 您好. 好的, 我帮您提交上去处理问题. 请您稍等我们几天.

Mike777Khoo,您好 核实清楚没有？ 已经快2周了； 处理不了， 就退款吧

**Reply by ArvieWang** [reply]
[Mike777Khoo](https://github.com/Mike777Khoo)@Mike777Khoo  何时能退款到账？
